back-office/multistoregeneralspecs.md

```diff
@@ -32,17 +38,35 @@ If the shop or the group selected doesn't have yet a color configured, the defau
 In this case, an information message is displayed on the page to prompt the user to define a color with a link opening the store's or group's page edition: "Customize your multistore header, follow the link to pick a color for this shop context" in Admin.Navigation.Header<br/>
 The user can close the information message. The message is closed only for the selected context and it will not be redisplayed if the user refreshes or returns to this page later.
 
+**User acceptance test**
+
+Given I am a merchant in a multistore context, when I am in a specific shop context in the back-office and if I have define a color for it, then I want to see the color of the shop selected in the header
+
+Given I am a merchant in a multistore context, when I am in a shop group context in the back-office and if I have define a color for it, then I want to see the color of the shop group selected in the header
+
+Given I am a merchant in a multistore context, when I am in "all shops" context in the back-office then I want to see the header in blue #25B9D8
+
+Given I am a merchant in a multistore context, when I am in a specific shop context or in a shop group context in the back-office and if I haven't define any color for it, then I want to see the header in light grey #EBEBEB
+
 **Name of the selected context**
 
 The name of the selected context is displayed, it can be:
 - All shops
 - A shop group name
 - A shop name
```

Comment:
Should we add that it can be added/edited in Advanced Parameters > Multistore?

Reply:
Yes, done :)